src/Post/Post.js

```diff
@@ -10,6 +8,41 @@ import { useIGData } from '../hooks/useIGData';
 import { absoluteToRelativeDate } from '../utils';
 import './Post.css';
 
+function PostLikeButton({
+  is_post_liked,
+  index,
+  toggleLike,
+  commentId,
+  replyId,
+  size = '24',
+  width = '28',
```

Comment:
Usually, it's either `size` alone, or `width` and `height`. Width and Size don't make much sense.